.packit.yaml

```diff
@@ -1,48 +1,50 @@
 ---
 # https://packit.dev/docs/configuration/
 
-specfile_path: fido-device-onboard.spec
+upstream_project_url: https://github.com/fdo-rs/fido-device-onboard-rs
+upstream_package_name: fido-device-onboard-rs
+upstream_tag_template: v{version}
 
-files_to_sync:
-    - fido-device-onboard.spec
-    - .packit.yaml
+issue_repository: https://github.com/fdo-rs/fido-device-onboard-rs
 
-upstream_package_name: fido-device-onboard
 downstream_package_name: fido-device-onboard
 
-upstream_tag_template: v{version}
 copy_upstream_release_description: true
 
+specfile_path: fido-device-onboard.spec
+files_to_sync:
+    - fido-device-onboard.spec
+    - .packit.yaml
+
 srpm_build_deps:
     - cargo
 
-actions:
-    create-archive:
-        - bash -c "sed -i -r  \"s/^Version:.*/Version:\        ${PACKIT_PROJECT_VERSION}/\" fido-device-onboard.spec"
-        - bash -c "sed -i '/Source1/d ; /^# See make-vendored-tarfile.sh in upstream repo/d ;' fido-device-onboard.spec"
-        - bash -c "cp ./patches/0001-Revert-chore-use-git-fork-for-aws-nitro-enclaves-cos.patch ."
-        - bash -c "git archive --prefix=fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}/ --format=tar HEAD > fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar"
-        - bash -c "tar -xvf fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar"
-        - bash -c "ls -1 ./fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar"
-    fix-spec-file:
-        - "cat fido-device-onboard.spec"
-
 jobs:
-    - job: copr_build
+    - &fdo_copr
+      job: copr_build
       trigger: pull_request
       targets:
-          - fedora-development-aarch64
           - fedora-development
+          - fedora-development-aarch64
           - fedora-latest
           - fedora-latest-aarch64
-    - job: copr_build
+          - centos-stream-9-x86_64
```

Comment:
Looks like we might be able to do EPEL builds for cs9?  re:  https://github.com/containers/podman/blob/main/.packit.yaml

Reply:
We could do scratch EPEL builds, but the EPEL rules are that packages can't duplicate what's in RHEL so we can't package it there because it'll conflict and customers end up in a mess!

Reply:
The rule makes sense, though I wonder how `podman` is getting around it.

For our purposes, it seems like skipping c9s and EPEL would be wise.

Reply:
I am not sure I am following... The centos-stream9 builds are just builds triggered in packit's copr not Koji or any other downstream copr/dist-git or whatever so I don't think we need any commit formats/approved bugs or anything else. The packit produced RPMs will be useful to run the tests on the specific platforms and that what we needed, right?.

On the other hand, I might be wrong but the EPEL builds @miabbott is referring to are just copr builds of CentOS Stream 9 with the EPEL-next repo activated, nothing more, the resulting RPMs are not going to be in EPEL-next, they just need EPEL-next to be built. As we don't need any EPEL repository to build the FDO packages I will leave centos-stream9 as it is.

Reply:
TBH, I'm not terribly familiar with how Packit handles c9s, so I was just looking for examples of other projects using it.

To that point, `osbuild` is using it for all downstream builds - https://github.com/osbuild/osbuild-composer/blob/main/.packit.yaml

And looking at their dashboard from Packit, shows that the c9s builds are just going to a copr - https://dashboard.packit.dev/projects/github.com/osbuild/osbuild

So I'll walk back my statement about skipping c9s...seems like it is something we can try.  (We'll still need to do manual work to get "official" c9s builds made...or invest in a bot to do it for us like the `osbuild` folks have)

Reply:
> I am not sure I am following... The centos-stream9 builds are just builds triggered in packit's copr not Koji or any 

So by centos-stream9 builds you're actually referring to copr builds against the c9s? I thought you meant as in c9s proper, when read "centos-stream9 builds" I read it as official c9s builds not copr el9 builds. Anything in copr is fine.